ecgtools/parsers.py

```diff
@@ -92,3 +102,208 @@ def cmip6_default_parser(
         data = {'exception': str(e), 'file': filepath}
         print(data)
         return {}
+
+
+class YAML_Parser:
```

Comment:
Can we use the recommended UpperCaseCamelCase convention here?

```suggestion
class YAMLParser:
```

This is not a big deal though

Reply:
Thanks @andersy005 , I went ahead and made this change.